### DISCUSSION TITLE
How to detect whether the websocket is connected successfully

### DISCUSSION THREAD

**Discussion by vic-x** [WalletConnect/walletconnect-monorepo#3172] [Developer Support]
When wss://relay.walletconnect.com/ is not connected successfully, calling web3modal.openModal() will report Error: Socket stalled when trying to connect to wss://relay.walletconnect.com, how can I get the error message

**Comment by jordangrant** [comment]
had this error moments ago seemingly fixed now

**Reply by glitch-txs** [reply]
you can try "catching" the init function if I'm not mistaken..
